packages/docusaurus-plugin-openapi-docs/src/types.ts

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  * ========================================================================== */
 
+import { SidebarItemDoc } from "@docusaurus/plugin-content-docs/src/sidebars/types";
```

Comment:
I'm not sure that this is the right path to import types from

Reply:
I think it works, since our plugin aims to be 100% compatible with Docusaurus sidebars, i.e. we don't extend the `SidebarItemDoc` type.